testbook/client.py

```diff
@@ -145,6 +145,36 @@ def cell_output_text(self, cell) -> str:
 
         return self._output_text(self.nb['cells'][cell_index])
 
+    def cell_execute_results(self, cell: Union[int, str]) -> List[Dict[str, Any]]:
```

Comment:
Maybe we should name the function `cell_execute_result` instead because `execute_result` is a [keyword](https://nbformat.readthedocs.io/en/latest/format_description.html#execute-result) in the Jupyter notebook JSON schema.

Reply:
Good suggestion, I'll do.